workspaces/diff-engine/src/lib.rs

```diff
@@ -1,12 +1,12 @@
 #![allow(dead_code, unused_imports, unused_variables)]
 
-mod events;
-mod interactions;
-mod projections;
-mod protos;
-mod queries;
-mod shapes;
-mod state;
+pub mod events;
+pub mod interactions;
+pub mod projections;
+pub mod protos;
+pub mod queries;
+pub mod shapes;
+pub mod state;
```

Comment:
We've very intentionally not made all internal modules public, as that grows the API surface area to whatever consumes this crate. Instead, we like to be intentional about what we expose and keep internals just that: internal.